ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -3635,10 +3691,84 @@ DRSM8X:
   - MONITORING
 
 DICM:
-  id: "4953043284228747776"
-  description: "Damper isolation control and monitoring."
+  description: "Damper isolation control and monitoring. "
   is_abstract: true
   uses:
    - supply_air_isolation_damper_closed_status
    - supply_air_isolation_damper_open_status
-   - supply_air_isolation_damper_command
\ No newline at end of file
+   - supply_air_isolation_damper_command
+
+UV:
+  description: "Ultraviolet lamp operation."
+  is_abstract: true
+  opt_uses:
+  - ultraviolet_lamp_control_mode
+  uses:
+  - ultraviolet_lamp_run_command
+  - ultraviolet_lamp_run_status
+
+H3X:
+  description: "Heater monitoring."
+  is_abstract: true
+  implements:
+  - MONITORING
+  uses:
+  - heater_run_status_1
+  - heater_run_status_2
+  - heater_run_status_3
+  - heater_run_command_1
+  - heater_run_command_2
+  - heater_run_command_3
+
+HS3X:
+  description: "Heater monitoring."
+  is_abstract: true
+  implements:
+  - MONITORING
+  uses:
+  - heater_run_status_1
+  - heater_run_status_2
+  - heater_run_status_3
+
+SAM:
+  description: "Supply air monitoring for pressure, temperature and flow rate."
+  is_abstract: true
+  implements:
+  - MONITORING
+  opt_uses:
+  - supply_air_static_pressure_sensor
+  uses:
+  - supply_air_flowrate_status
```

Comment:
Description says flowrate, not flowrate status. Is this supposed to be a status?

Reply:
Change the description to flow rate status.